babel/messages/frontend.py

```diff
@@ -313,6 +313,8 @@ class extract_messages(Command):
          'files or directories with commas(,)'),  # TODO: Support repetition of this argument
         ('input-dirs=', None,  # TODO (3.x): Remove me.
          'alias for input-paths (does allow files as well as directories).'),
+        ('ignore-dir-prefixes=', None,
+          'Bypass directories whose name start with PREFIX. You can specify this more than once.'),
```

Comment:
How is specifying this more than once handled? All I see is attempting to split a string by `,`.